### PR TITLE
Added Chiniese language

### PR DESCRIPTION
**Pull request by 54853315**
Added Chiniese language

### CODE REVIEW

File: i18n.php [hunk 1]
```diff
@@ -264,4 +264,21 @@
         'undefined_count'         => '{1,21,31} %count% не определен|]1,Inf] %count% не определено',
         'skipped_count'           => '{1,21,31} %count% пропущен|]1,Inf] %count% пропущено',
     ),
+    'cn'    => array(
```
Comment (ciaranmcnulty) [inline i18n.php:267]: Isn’t it `zh`?

Comment (54853315) [reply i18n.php:267]: `zh` will be nice ,I will fix it .

Comment (ciaranmcnulty) [reply i18n.php]: I don’t know enough to say if it’s correct :)

You could consider contributing gherkin translations to cucumber/cucumber too

**Comment by ciaranmcnulty** [comment]
It would be good to get another Chinese speaker (reader?) to check this and confirm the translations are ok

**Comment by ciaranmcnulty** [comment]
Thank you!

**Pull request merged**